entrypoint.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
```

Comment:
can we add tests for this? similar to https://github.com/aquasecurity/trivy-action/blob/master/test/test.bats